graphgrid_sdk/ggcore/sdk_messages.py

```diff
@@ -314,3 +314,16 @@ class TrainRequestBody:
     def to_json(self):
         """Encode TrainRequestBody to a json object"""
         return json.dumps(self.__dict__, indent=4)
+
+
+class GetActiveModelResponse(SdkServiceResponse):
+    """Define class representing a get active model api call response."""
+    task: str
```

Comment:
lets remove this unused `task` field and replace with `modelName: str` and `trainedModelData: dict`